base/sparse/sparsematrix.jl

```diff
@@ -2565,6 +2678,15 @@ function spdiagm_internal(B, d)
     return (I,J,V)
 end
 
+doc"""
```

Comment:
Doesn't seem to need `doc"` prefix here. Probably best to keep things consistent and only use `doc"` when necessary.